### ISSUE TITLE
onPrivateSubscribe "cb" arguments does not match type declaration

### ISSUE THREAD

**Issue by hastom** [centrifugal/centrifuge-js#94]
Centrifuge v2.2.2

According to declaration it should take one argument of interface `SubscribePrivateResponse` which has 2 props status and channels. So something like this should be valid:
```
...
onPrivateSubscribe: async ({ data }, cb) => {
  const { channels } = await getWsChannelTokens(data);
    cb({
      status: 200,
      channels: channels,
  });
},
```
But it will result in error `TypeError: Cannot read property 'channels' of undefined` because the actual thing its waiting for is
```
{
    status: 200,
    data: { channels: channels },
}
```
So I guess data property is missing in type declarations

**Comment by FZambia** [comment]
@hastom hello, thanks for report. I will take a look soon - a bit busy these days.

**Comment by FZambia** [comment]
Fixed in 7126410b32b666de59c830e767668bd2ce39cb64. In general current format is not beautiful at all as it requires passing status 200 to work which is a bit awkward, we should keep backwards compatibility for now, but in future releases I believe it needs to be improved.

**Comment by hastom** [comment]
Nice!
